data/locale/en-US.ini

```diff
@@ -1,54 +1,59 @@
+# Note to translators: *DO NOT* translate this file directly.
+# Instead, visit http://crowdin.com/project/obs-ndi and submit your translations there.
+# Pull requests for translations outside of Crowdin will not be accepted.
+
 NDIPlugin.Default="Default"
-NDIPlugin.NDISourceName="NDI™ Source"
+NDIPlugin.NDISourceName="NDI® Source"
 NDIPlugin.SourceProps.SourceName="Source name"
 NDIPlugin.SourceProps.Bandwidth="Bandwidth"
+NDIPlugin.SourceProps.Bandwidth.Highest="Highest"
+NDIPlugin.SourceProps.Bandwidth.Lowest="Lowest"
+NDIPlugin.SourceProps.Bandwidth.AudioOnly="Audio Only"
 NDIPlugin.SourceProps.Behavior="Behavior"
 NDIPlugin.SourceProps.Behavior.Keep="Keep source connected"
 NDIPlugin.SourceProps.Behavior.Disconnect="Disconnect source when not visible"
 NDIPlugin.SourceProps.BehaviorLastFrame="Keep last frame when disconnected"
-NDIPlugin.SourceProps.Sync="Audio/Video Sync"
-NDIPlugin.NDIFrameSync="Framesync (experimental)"
-NDIPlugin.SourceProps.HWAccel="Allow hardware acceleration"
+NDIPlugin.SourceProps.Sync="Synchronize Audio/Video"
+NDIPlugin.SourceProps.Sync.NDITimestamp="Network"
+NDIPlugin.SourceProps.Sync.NDISourceTimecode="Source Timing"
+NDIPlugin.SourceProps.NDIFrameSync="NDI® Framesync (experimental)"
+NDIPlugin.SourceProps.HWAccel="Request hardware acceleration"
 NDIPlugin.SourceProps.AlphaBlendingFix="Fix alpha blending (adds a filter to this source)"
 NDIPlugin.SourceProps.ColorRange="YUV Range"
 NDIPlugin.SourceProps.ColorRange.Partial="Limited"
 NDIPlugin.SourceProps.ColorRange.Full="Full"
 NDIPlugin.SourceProps.ColorSpace="YUV Color Space"
-NDIPlugin.SourceProps.Latency="Latency Mode"
-NDIPlugin.SourceProps.Latency.Normal="Normal (safe)"
-NDIPlugin.SourceProps.Latency.Low="Low"
-NDIPlugin.SourceProps.Latency.Lowest="Lowest (unbuffered)"
+NDIPlugin.SourceProps.LatencyMode="Latency Mode"
+NDIPlugin.SourceProps.LatencyMode.Normal="Normal (safest/recommended)"
+NDIPlugin.SourceProps.LatencyMode.Low="Low"
+NDIPlugin.SourceProps.LatencyMode.Lowest="Lowest (unbuffered; transparent background not supported)"
```

Comment:
Could be confusing (it is only transparency in background or ANY transparency or anything related to alpha channel (which provide the transparency.
Suggestion :
Alpha channel not supported

Later we might have to add that HDR to this list.